kss-node/kss-node
# 164: Allow templates to have a configuration file

- [x] Allow yargs to take multiple variables
- [x] Figure out how to generator to read and recognize options from configuration file 
- [x] Added default values if no variable is specified

Steps to test: 
create a config file (config.json) in directory with following keys

{
  "source": "sass",
  "destination": "styleguide",
  "title": "Test Title"
}

run ../kss-node/bin/kss-node --config config.json < with checked-out 164-template-config

styleguide should generate with specified title

change config to omit title, default value
{
  "source": "sass",
  "destination": "styleguide"
}

styleguide should generate with default title
